testing.py

```diff
@@ -1,18 +1,32 @@
-#!/bin/env python
-"""
-Script to take Scurve data using OH ultra scans
-By: Cameron Bravo (c.bravo@cern.ch)
-"""
+#!/usr/bin/env python
```

Comment:
Actually, as I requested more changes, why don't you make the change and include it in your update